### PR TITLE
Cypress/fe 1670

### PR DESCRIPTION
**Pull request by marcinstachowicz**
# cypress maintenance

## update readme file

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -24,21 +24,26 @@ Use scenario tags:
 3. `@ignore` for temporary ignored scenarios
 4. `@[bug-number]` for example `@FE-1234` - use this tag after `@ignore` to explain why test is ignored.
 5. `@ignore` and `@[bug-number]` should be removed after bug fix
+6. `@build` for pipeline - each build is veirfied by this tests
```
Comment (DlgSHi) [inline cypress/README.md]: for a pipeline - each build will be verified by those tests :)

File: cypress/README.md [hunk 2]
```diff
@@ -24,21 +24,26 @@ Use scenario tags:
...
 
 ## Files structure
 ```
 .
 ├── cypress
 │ ├── fixture
-│ ├── integration
+│ ├── features
+│ │   ├── build
+│ │   ├── sanity
```
Comment (DlgSHi) [inline cypress/README.md]: sanity package is under regression :)

Comment (marcinstachowicz) [reply cypress/README.md]: I use this order to show status of tests:
build/pipeline - build folder
sanity - Accessibility section - more than build
regression - all other tests

File: cypress/README.md (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/README.md]: today will be, but tomorrow... who knows

**Pull request merged**